### ISSUE TITLE
Recompute uvw during averaging

### ISSUE THREAD

**Issue by landmanbester** [ratt-ru/pfb-imaging#112]
The init app has options to do simple averaging in frequency and time based BDA using the averaging functionality in [codex-africanus](https://github.com/ratt-ru/codex-africanus/tree/master/africanus/averaging). If I understand correctly, these do not recompute the uvw coordinates from the averaged time and frequency centroids but approximates them with a Euclidean average (is this correct or is it a weighted average @sjperkins @bennahugo?). We should probably rather recompute them as is done in [xova](https://github.com/ratt-ru/xova)

**Comment by sjperkins** [comment]
Yes it is a weighted average and technically the UVW coordinates should be recomputed.

**Comment by sjperkins** [comment]
In general this is something that I'd like to implement in arcae, but that will have to wait until the data processing churn has quietened down.

**Comment by landmanbester** [comment]
From my tests it doesn't seem to make a big difference if you are doing a moderate amount of averaging. At least for the L-band MeerKAT data I have been working with setting `-max-fov 2.2` and using a `bda-dorr 0.98` I don't see any difference in the 2 degree fov I am reconstructing and yet I get a factor of about 5 in compression. Big win so thanks. Low priority but thought I would document it for posterity
